src/generic/alert-message/index.scss

```diff
@@ -0,0 +1,6 @@
+// TODO: remove this after upstream fix merging: https://github.com/openedx/paragon/pull/3562
+.alert {
+  .alert-message-content {
+    align-self: baseline;
+  }
+}
```

Comment:
With this it both the message and icon are not cenetered.
![image](https://github.com/user-attachments/assets/25058cd1-f99d-4eb8-8a1c-b8f36ca748bc)

Also I see the icon a few pixels lower
![image](https://github.com/user-attachments/assets/ef222c6e-297e-4e18-81ca-c2ce7f123282)

if you instead center the icon, everthing will be aligned correctly

```suggestion
.alert {
  .alert-icon {
    align-self: center;
  }
}
```
![image](https://github.com/user-attachments/assets/c8571578-172e-4611-bba0-5c61aed7652c)

![image](https://github.com/user-attachments/assets/a5d9de64-8c50-411a-8af1-ae4b8af1237b)

Reply:
you are right!